libcudacxx/include/cuda/std/bitset

```diff
@@ -0,0 +1,22 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _CUDA_STD_BITSET
+#define _CUDA_STD_BITSET
+
+#include <cuda/std/detail/__config>
+
+_CCCL_PUSH_MACROS
+
+#include <cuda/std/detail/libcxx/include/bitset>
```

Comment:
Question: Can we inline everything here, so that we can drop `cuda/std/detail/libcxx/include/bitset`